htdocs/landing/aup.html

```diff
@@ -0,0 +1,104 @@
+          <li><p>You shall provide appropriate acknowledgement of support or citation for your use of the resources/services provided as required by the body or bodies granting you access.</p></li>
+          <li><p>You shall not use the resources/services for any purpose that is unlawful and not (attempt to) breach or circumvent any administrative or security controls.</p></li>
+          <li><p>You shall respect intellectual property and confidentiality agreements.</p></li>
+          <li><p>You shall protect your access credentials (e.g. private keys or passwords).</p></li>
+          <li><p>You shall keep all your registered information correct and up to date.</p></li>
+          <li><p>You shall immediately report any known or suspected security breach or misuse of the resources/services or access credentials to the specified incident reporting locations and to the relevant credential issuing authorities.</p></li>
+          <li><p>You use the resources/services at your own risk. There is no guarantee that the resources/services will be available at any time or that their integrity or confidentiality will be preserved or that they will suit any purpose.</p></li>
+          <li><p>You agree that logged information, including personal data provided by you for registration purposes, may be used for administrative, operational, accounting, monitoring and security purposes. You agree that this logged information may be disclosed to other authorised participants via secured mechanisms, only for the same purposes and only as far as necessary to provide the services.</p></li>
+          <li><p>You agree that the body or bodies granting you access and resource/service providers are entitled to regulate, suspend or terminate your access without prior notice and without compensation, within their domain of authority, and you shall immediately comply with their instructions.</p></li>
+          <li><p>You are liable for the consequences of your violation of any of these conditions of use, which may include but are not limited to the reporting of your violation to your home institute and, if the activities are thought to be illegal, to appropriate law enforcement agencies.</p></li>
+        </ol>
+        <h1 id="references">References</h1>
+        <table>
+          <thead>
+            <tr class="header">
+              <td>R 1</td>
+              <td>(Old version) Grid Acceptable Use Policy: <a href="https://documents.egi.eu/document/74" class="uri">https://documents.egi.eu/document/74</a></td>
+            </tr>
+          </thead>
+          <tbody>
+            <tr class="odd">
+              <td>R 2</td>
+              <td>Approved EGI Security Policies: <a href="https://wiki.egi.eu/wiki/SPG:Documents" class="uri">https://wiki.egi.eu/wiki/SPG:Documents</a></td>
+            </tr>
+          </tbody>
+        </table>
+        <br>
+        <table>
```

Comment:
This section needs a heading. It looks like part of the references at the moment.

Reply:
Accepted. Headed as "Document Information".